src/core/channel/client_setup.c

```diff
@@ -148,7 +180,8 @@ int grpc_client_setup_cb_begin(grpc_client_setup_request *r) {
   return 1;
 }
 
-void grpc_client_setup_cb_end(grpc_client_setup_request *r) {
+void grpc_client_setup_cb_end(grpc_client_setup_request *r,
+                              const char *reason) {
```

Comment:
It was there for some debugging getting everything up. Might be useful for debugging in the future. Then again, this module is scheduled for a rewrite next week to support the naming/load balancing efforts. Ok with leaving it til then?